addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/order_receipt.xml

```diff
@@ -3,115 +3,116 @@
     <t t-name="point_of_sale.OrderReceipt">
         <div class="pos-receipt p-2">
             <t t-set="showTaxGroupLabels" t-value="doesAnyOrderlineHaveTaxLabel()"/>
-            <ReceiptHeader data="props.data.headerData" />
-            <OrderWidget lines="props.data.orderlines" t-slot-scope="scope"
-                generalCustomerNote="props.data.general_customer_note or ''"
-                internalNote="props.data.internalNote or ''" screenName="props.data.screenName">
+            <ReceiptHeader data="this.header" order="order" />
```

Comment:
```suggestion
            <ReceiptHeader order="order" />
```
Like you do with the receipt, compute the header data from the ReceiptHeader component using the given order.